socorro/processor/processor_app.py

```diff
@@ -131,6 +180,10 @@ class ProcessorApp(FetchTransformSaveApp):
         from_string_converter=class_converter,
     )
 
+    def __init__(self, config):
+        super().__init__(config)
+        self.waiting_func = None
```

Comment:
I think we can get rid of the `waiting_func`. I'm not sure what it does and it appears to always be set to `None`.